### PR TITLE
docs(design): fix typo

### PR DESCRIPTION
**Pull request by MarcoIeni**

### CODE REVIEW

File: docs/design.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Bors design
-This document briefly describes does the bors bot work.
+This document briefly describes how the bors bot work.
 
-`bors` is a binary that launches (an `axum`) web server, which listens on a `/github` endpoint for webhooks related to a
+`bors` is a binary that launches an `axum` web server, which listens on a `/github` endpoint for webhooks related to a
```
Comment (MarcoIeni) [inline docs/design.md:4]: I think the phrase should make sense even without the parenthesis

**Comment by Kobzol** [comment]
It should be "how does" or "how ... works" :)

File: docs/design.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Bors design
-This document briefly describes does the bors bot work.
+This document briefly describes how the bors bot work.
```
Comment (MarcoIeni) [inline docs/design.md:2]:
```suggestion
This document briefly describes how the bors bot works.
```

**Review by Kobzol — APPROVED** [review]
Thanks :)

**Pull request merged**